recipes/mathter/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
```

`cxx_standard_17` was added later

```suggestion
cmake_minimum_required(VERSION 3.8)
```